### DISCUSSION TITLE
Abertura das Discussões :tada:

### DISCUSSION THREAD

**Discussion by heyitsmepablo** [TheDevOpsCorp/projeto-final-trainee-backend#10] [Announcements]
Olá pessoal, estou testando o ambiente de discurssão do nosso repositorio, visando o aprendizado e a maior interação com todos a abertura está diretamente relacionada ao repositorio projeto final trainee

# Regras

### 5. Pode zueiras e brincadeiras desde não desacate a regra 1 👍

**Comment by MrErykCardoso** [comment]
Muito bom saber que temos espaço para esse tipo de compartilhamento por aqui! Postarei meus questionamentos por aqui também agora.

**Reply by heyitsmepablo** [reply]
Sim sim, muito legal !

**Comment by ItzTas** [comment]
isso é tipo um discord do github?

**Reply by heyitsmepablo** [reply]
Ta mais pra um forum

**Comment by ed-henrique** [comment]
**TLDR** Bora usar esse recurso ao invés de reuniões quando possível, pra ficar registrado o que foi discutido?

Não sabia que dava pra criar uma discussão na raiz da organização, achava que era só nos repositórios mesmo. Tava vendo aqui, alguns usos comuns pra essas discussões são:

- Anúncio de release novo;
- Anúncio de novidades, blogs e artigos que envolvam o produto;
- Q&A pra responder dúvidas sobre um produto;
- Decidir a direção de desenvolvimento de um produto com os devs e usuários.

Acho que a gente podia criar essas discussões nos repositórios específicos quando for precisar discutir alguma coisa específica que vá afetar a aplicação, tipo modelagem do banco de dados.

Fazendo isso a gente evita o que acontecia no Stockee lá no início, que a gente tomava uma decisão, aplicava, e em uma semana ninguém sabia mais o que tinha feito kkk.

Dava de usar isso ao invés de coletar as atas das reuniões pra ficar registrando o que foi discutido, o que vocês acham? Talvez dessa forma tenham até menos reuniões, porque elas iam ser só para coisas mais urgentes.

**Reply by MrErykCardoso** [reply]
Parece uma boa ideia, apesar das reuniões presenciais serem mais dinâmicas. Mas acho que nada impede de que as discussões por aqui compram esse papel. As falas podem até ser mais elaboradas já que aqui tem bastante recursos de comunicação, tanto de código, quanto imagem e anexos em geral.
